tests/testSproc-check_assessment_access.js

```diff
@@ -161,6 +161,42 @@ describe('sproc check_assessment_access* tests', function() {
             });
         });
 
+        it('pass if all parameter match match except UID with no uids', function(callback) {
```

Comment:
I think you mean `pass if all parameters match except UID with no uids`

Reply:
fixed